renku/core/commands/rollback.py

```diff
@@ -0,0 +1,294 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018-2021- Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Renku ``status`` command."""
+
+
+import os.path
+import re
+from itertools import islice
+
+from renku.core.management.command_builder import inject
+from renku.core.management.command_builder.command import Command
+from renku.core.management.interface.client_dispatcher import IClientDispatcher
+from renku.core.management.interface.database_dispatcher import IDatabaseDispatcher
+from renku.core.models.dataset import Dataset
+from renku.core.models.provenance.activity import Activity
+from renku.core.models.workflow.plan import AbstractPlan
+from renku.core.utils import communication
+
+CHECKPOINTS_PER_PAGE = 50
```

Comment:
why define twice this static var? see in `renku/cli/rollback.py` above `CHECKPOINTS_PER_PAGE = 10`